packages/terra-paginator/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixed cosmetic effects of text-wrapping
```

Comment:
Can we add in a test case to show these fixes?

Reply:
The effect was not visible for the widths which we use for wdio tests.So, I had not added the test

Reply:
Can you provide a container with a set width that *does* cause the issue?

Reply:
Added wdio test.